generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -285,10 +285,16 @@ public class <%= asEntity(entityClass) %> implements Serializable {
 }
 
 relationships.forEach((relationship, idx) => {
-    const otherEntityRelationshipName = relationships[idx].otherEntityRelationshipName;
+    const otherEntityRelationshipName =
+        databaseType === 'neo4j'
+            ? "HAS_" + _.toUpper(_.snakeCase(relationships[idx].otherEntityRelationshipName))
+            : relationships[idx].otherEntityRelationshipName;
     const otherEntityRelationshipNamePlural = relationships[idx].otherEntityRelationshipNamePlural;
     const otherEntityIsEmbedded = relationships[idx].otherEntityIsEmbedded;
-    const relationshipName = relationships[idx].relationshipName;
+    const relationshipName =
```

Comment:
If there is a different nomenclature for relationship name, the rule should be applied where it is calculated?